File: data/src/main/java/data/models/Inspiration.java
package data.models;

import java.util.ArrayList;

public class Inspiration {
	
	private City city;
	private ArrayList<Activity> activities = new ArrayList<Activity>();
	private Weather weather;
	private ArrayList<Photo> photos = new ArrayList<Photo>();
	
	public Inspiration() {
		
	}
	
	public Inspiration(City city, ArrayList<Activity> activities, Weather weather, ArrayList<Photo> photos) {
		this.city = city;
		this.activities = activities;
		this.weather = weather;
		this.photos = photos;
	}

	public City getCity() {
		return city;
	}

	public void setCity(City city) {
		this.city = city;
	}

	public ArrayList<Activity> getActivities() {
		return activities;
	}

	public void setActivities(ArrayList<Activity> activities) {
		this.activities = activities;
	}

	public Weather getWeather() {
		return weather;
	}

	public void setWeather(Weather weather) {
		this.weather = weather;
	}

	public ArrayList<Photo> getPhotos() {
		return photos;
	}

	public void setPhotos(ArrayList<Photo> photos) {
		this.photos = photos;
	}

}

File: data/src/angular-client/src/app/models/inspiration.ts
import {Injectable} from '@angular/core';

import { Activity } from './../models/activity.js';
import { City } from './../models/city.js';
import { Photo } from './../models/photo.js';
import { Weather } from './../models/weather.js';

@Injectable()
export class Inspiration {
  public city: any;
  public photos: any;
  public weather: any;
  public activities: any;
  public some = "link";
  
//  constructor(city: {}) {
////    this.city = new City(city["id"], city["name"], city["history"], city["description"]);
////    
////    for(let p in photos) {
////      const photo = new Photo(p["id"], p["cityId"], p[this.some]);
////      this.photos.push(photo);
////    }
////    
////    this.weather = new Weather(weather["temperatures"]["id"], weather["temperatures"]["cityId"], weather["temperatures"]["values"]);
////    
////    for(let a in activities) {
////      const activity = new Activity(a["id"], a["cityId"], a["city"], a["description"], a["photoLink"], a["address"]);
////      this.activities.push(activity);
////    }
//    
//  }
  
}